build.sh

```diff
@@ -71,7 +71,7 @@ install_rpms() {
 
     # Add fast-tracked packages here.  We don't want to wait on bodhi for rpm-ostree
     # as we want to enable fast iteration there.
-    yum -y --enablerepo=updates-testing upgrade rpm-ostree
+    yum -y --enablerepo=updates-testing upgrade rpm-ostree virtiofsd
```

Comment:
The update is in the stable repos now. 
```suggestion
    yum -y --enablerepo=updates-testing upgrade rpm-ostree
```